lib/src/loadable_list_view.dart

```diff
@@ -29,6 +37,10 @@ class LoadableListViewState<T extends StoreListItem>
     if (viewModel?.loadList != null && viewModel.loadListRequestState.isIdle) {
       viewModel?.loadList();
     }
+
+    scrollController.addListener(() {
+      onChangeContentOffset?.call(scrollController.position.pixels);
```

Comment:
It's better to use `widget?.onChangeContentOffset?.call(scrollController.position.pixels);`
I think.